dev/com.ibm.ws.session.cache_fat/test-applications/multiServerApp/resources/META-INF/permissions.xml

```diff
@@ -0,0 +1,56 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Is it really necessary to duplicate all of the application files?  Seems like we could have a single copy of the application and put it onto all of the servers.  I realize that in one case you want to include testing of a data source and in the other case not, but if that's the only difference, that could be done with 2 different servlet methods rather than duplicating everything.